scripts/irods/test/test_itree.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function
+
+import sys
+
+from . import session
+from .. import test
+
+if sys.version_info < (2,7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+
+class Test_ITree(session.make_sessions_mixin([('otherrods', 'rods')], []),
+                 unittest.TestCase):
+
+    def setUp(self):
+        super(Test_ITree, self).setUp()
+        self.admin = self.admin_sessions[0]
+
+    def tearDown(self):
+        super(Test_ITree, self).tearDown()
+
+    @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, "Skip for Topology Testing")
```

Comment:
i don't know that any of these should behave differently in a topology configuration... do we need this skip? or the other three below?

Reply:
Not really, but it's what the other icommand tests seem to do.

Reply:
Not to beat the dead horse, but it looks to me like this would behave fine in a topology test setup.

Reply:
oh, missed that - yes please remove this one too.